2199/2/versus.rb

```diff
@@ -0,0 +1,141 @@
+require 'yaml'
+require 'bundler'
+Bundler.require
+
+# :reek:InstanceVariableAssumption
+# class Battle to collect information
+class Battle
+  def initialize(filename)
+    @filename = filename
+  end
+
+  def bad_words_count
+    words.count do |word|
```

Comment:
This may be a time-consuming operation. Can we do something about it?

Reply:
I don't know..